public/style.css

```diff
@@ -102,9 +102,10 @@ filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#009564",endCo
   width: 80px;
   height: 80px;
   margin: auto;
-  background: black;
+  /* background: black; */
```

Comment:
Let's just start removing dead CSS, we have too much. We can always retrieve old CSS from previous commits.